src/commands/cat.rs

```diff
@@ -0,0 +1,61 @@
+use std::{path::PathBuf, str::FromStr};
+
+use tokio::io::{stdout, AsyncWriteExt};
+use vex_v5_serial::{
+    commands::file::DownloadFile,
+    connection::{
+        serial::{SerialConnection, SerialError},
+        Connection,
+    },
+    packets::file::{FileTransferTarget, FileVendor},
+    string::FixedString,
+};
+
+use crate::errors::CliError;
+
+pub fn vendor_from_prefix(prefix: &str) -> FileVendor {
+    match prefix {
+        "user" | "/user" => FileVendor::User,
+        "sys_" | "/sys_" => FileVendor::Sys,
+        "rmsh" | "/rmsh" => FileVendor::Dev1,
+        "pros" | "/pros" => FileVendor::Dev2,
+        "mwrk" | "/mwrk" => FileVendor::Dev3,
+        "deva" | "/deva" => FileVendor::Dev4,
+        "devb" | "/devb" => FileVendor::Dev5,
+        "devc" | "/devc" => FileVendor::Dev6,
+        "vxvm" | "/vxvm" => FileVendor::VexVm,
+        "vex_" | "/vex_" => FileVendor::Vex,
+        _ => FileVendor::Undefined,
+    }
+}
+
+pub async fn cat(connection: &mut SerialConnection, file: PathBuf) -> Result<(), CliError> {
+    let vendor = vendor_from_prefix(if let Some(parent) = file.parent() {
+        parent.to_str().unwrap()
+    } else {
+        ""
+    });
```

Comment:
```suggestion
    let vendor = if let Some(parent) = file.parent() {
        vendor_from_prefix(parent.to_str().unwrap())
    } else {
        FileVendor::Undefined
    });
```

Reply:
This will result in a path being concatenated to `test/{}`, which I don't think is the behavior we want.

Reply:
Actually nevermind, this is just semantics since we strip the VID folder name before passing to VEXos anyways. I'll make the change since the proposed code is cleaner.